modules/clients/src/test/java/org/apache/ignite/common/RunningQueryInfoCheckInitiatorTest.java

```diff
@@ -227,7 +227,7 @@ public void testJdbcV2InitiatorId() throws Exception {
 
             GridTestUtils.runAsync(() -> {
                     try (Connection conn = DriverManager.getConnection(
-                        CFG_URL_PREFIX + "nodeId=" + grid0NodeId + "@modules/clients/src/test/config/jdbc-config.xml")) {
+                        CFG_URL_PREFIX + "nodeId=" + grid0NodeId + "@modules/clients/src/test/config/jdbc-security-config.xml")) {
```

Comment:
Because only this test checks JDBC connection to the cluster for which Ignite authentication is enabled through Ignite node.
As were mentioned in the dev list discussion:
"The current implementation of IgniteAuthenticationProcessor can enable authentication itself if the current node connects to the cluster with authentication enabled - this functionality will not be supported. The user can easily overcome this by explicitly enabling authentication in the configuration on all nodes."

So in this test case with respect to IgniteAuthenticationProcessor changes by this PR we must explicitly enable authentication for the node that JDBC uses to connect to the cluster.

That was my motivation to create the new configuration file.

This test class is all about security but previously it works because the authentication on joining node was enabled automatically.

Reply:
Got it. Thanks for the clarification.